src/main/java/core/basesyntax/service/FilesWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FilesWriter {
+    void write(String fileName);
```

Comment:
```suggestion
    void write(Map<Fruit, Integer> report, String fileName);
```
let's communicate with storage only in FruitService

Reply:
fixed